oracle/provider/binance.go

```diff
@@ -125,7 +125,12 @@ func NewBinanceProvider(
 		subscribedPairs: map[string]types.CurrencyPair{},
 	}
 
-	confirmedPairs, err := provider.ConfirmPairAvailability(pairs...)
+	confirmedPairs, err := ConfirmPairAvailability(
+		provider,
+		provider.endpoints.Name,
```

Comment:
Do you need to pass these in separately if you are passing the provider object in?

Reply:
Yeah because they're declared in the actual provider struct (i.e. BinanceProvider) and not the provider interface